app/client/src/pages/VerifyYourSeedPhrase/VerifyYourSeedPhrase.jsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+import {Button, Container, Grid, Header, TextArea} from "semantic-ui-react";
+
+import {useHistory} from "react-router-dom";
+
+import HeaderMenu from "../../Components/HeaderMenu";
+
+function VerifyYourSeedPhrase({seedPhrase}) {
+
+    const [seedPhraseIsCorrect, setSeedPhraseIsCorrect] = React.useState(false);
+
+    const [chosenPhrase, setChosenPhrase] = React.useState([]);
+
+    const [verifyPhraseButtonText, setVerifyPhraseButtonText] = React.useState("Verify Phrase");
+
+    const history = useHistory();
+
+    React.useEffect(() => {
+        if (seedPhraseIsCorrect) {
+            setVerifyPhraseButtonText("Verify Phrase")
+        } else {
+            setVerifyPhraseButtonText("Phrase isn't correct yet!")
+        }
+    }, [seedPhraseIsCorrect]);
+
+    React.useEffect(() => {
+        setSeedPhraseIsCorrect(chosenPhrase.length === 12);
```

Comment:
Good pluggable code, can easily update to correct verification functions when written :star: